src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,34 @@
+(function(transport, console) {
+    "use strict";
+    
+    function transportSend(type, args) {  
+        var data = {args: args, type: type};
+        transport.send("bramble-console", data);
+    }
+    
+	// Implement standard console.* functions
+	["log"
+        , "warn"
```

Comment:
This is a style thing, but I'm not a fan of using a leading comma.  Can you please put it at the end of each item?

```js
["log",
  "warn",
```